src/screens/ProfileScreen/index.js

```diff
@@ -1,6 +1,8 @@
 import * as React from 'react';
 import ImagePicker from 'react-native-image-crop-picker';
+import PropTypes from 'prop-types';
 import Toast from 'react-native-simple-toast';
+import netInfo from '@react-native-community/netinfo';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` and `netInfo` has been removed. Ensure that these are not used anywhere else in the file, otherwise it will cause a reference error.
```diff
- import PropTypes from 'prop-types';
- import netInfo from '@react-native-community/netinfo';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->